### ISSUE TITLE
Address SonarCloud issues

### ISSUE THREAD

**Issue by dougwaldron** [gaepdit/template-app#34]
[template-app project on SonarCloud](https://sonarcloud.io/project/overview?id=gaepdit_template-app)

**Comment by dougwaldron** [comment]
A lot of the issues identified by SonarCloud are caused by library files. Work on this issue should be postponed until after #4 is fixed.
